charts/urbanstack-shared-app/templates/networkpolicy.yaml

```
@@ -0,0 +1,59 @@
+  name: {{ .Release.Name | quote }}
+  labels: {{- include "urbanstack-shared-app.labels" . | nindent 4 }}
+spec:
+  podSelector:
+    matchLabels:
+      {{- include "urbanstack-shared-app.selectorLabels" . | nindent 6 }}
+  policyTypes:
+    - Egress
+    - Ingress
+  egress:
+    {{- if .Values.allowExternalEgress }}
+    - to: # allow external endpoints outside of the cluster
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)